worker/src/sidechain_storage/error.rs

```diff
@@ -0,0 +1,25 @@
+/*
+	Copyright 2019 Supercomputing Systems AG
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+		http://www.apache.org/licenses/LICENSE-2.0
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+*/
+use substratee_node_primitives::ShardIdentifier;
+
+pub type Result<T> = std::result::Result<T, Error>;
+
+#[derive(Debug)]
```

Comment:
Fixed. Only [one .map_err](https://github.com/integritee-network/worker/blob/sidechain-storage/worker/src/sidechain_storage/storage.rs#L154) left that seems to be enforced by cargo.

Reply:
Nice! Yes, this is because without the `?` rust does not perform the implicit `into` conversion. So, you either do it your way, or do `Ok(db.write()?)`

Reply:
I'll leave it this way, seems to be more straight forward than unwraping and wrapping again.